assignment2/src/com/company/main/Details.java

```diff
@@ -0,0 +1,150 @@
+package com.company.main;
+
+import java.io.Serializable;
+import java.util.*;
+import java.io.*;
+
+public class Details implements Serializable {
+    String name, addr, courses;
+    String[] coursesSplit ;
+    int age, rollNo;
+    static List<Student> students = new ArrayList<>();
+
+    Student st = new Student();
+
+    Scanner input = new Scanner(System.in);
+    public void input()
+    {
```

Comment:
don't have functions with names that don't give enough context as to what they do... input() but input of what ... input to do what ??